queries/ne.jinja2

```diff
@@ -1,4 +1,4 @@
-{% set ne_languages = ['ar', 'bn', 'de', 'en', 'es', 'fr', 'el', 'hi', 'hu', 'id', 'it', 'ja', 'ko', 'nl', 'pl', 'pt', 'ru', 'sv', 'tr', 'vi', 'zh'] %}
+{% set ne_languages = ['ar', 'bn', 'de', 'el', 'en', 'es', 'fa', 'fr', 'he', 'hi', 'hu', 'id', 'it', 'ja', 'ko', 'nl', 'pl', 'pt', 'ru', 'sv', 'tr', 'uk', 'ur', 'vi', 'zh', 'zht'] %}
```

Comment:
`zht` is the critical one, but the new NE data also includes explicate English (instead of mostly English `name` singular), Farsi, Hebrew, Ukrainian, and Urdu.

Reply:
this means our integration test misses this path?

Reply:
The integration tests are based on "data fixtures" where you mock the **output** from a database query. 

So yes, they are missing this path. This is why we should **also** have (eg add) "build release tests" to make sure tiles have the right content for a full end-to-end prod promotion.